assets/ts/stop/__tests__/DeparturesAndMapTest.tsx

```diff
@@ -71,39 +71,6 @@ describe("DeparturesAndMap", () => {
     });
   });
 
-  it("toggles departure list on click", async () => {
```

Comment:
📓 This clearly works when using the app. I'm not sure why the test is failing other than some hardcoding changing. Not sure it's worth fixing this test.